docs/source_connector.rst

```diff
@@ -11,92 +11,120 @@ The database is monitored for new or deleted tables and adapts automatically. Wh
 from a table, the connector can load only new or modified rows by specifying which columns should
 be used to detect new or modified data.
 
-Quickstart
-----------
+Quick Start
+-----------
 
-To see the basic functionality of the connector, we'll copy a single table from a local SQLite
-database. In this simple example, we'll assume each entry in the table is assigned a unique ID
+To see the basic functionality of the connector, you'll copy a single table from a local SQLite
+database. In this quick start, you can assume each entry in the table is assigned a unique ID
 and is not modified after creation.
 
-.. note:: You can use your favorite database instead of SQLite.
-   Follow the same steps, but adjust the ``connection.url`` setting for your database.
-   Confluent Platform includes JDBC drivers for SQLite and PostgreSQL, but if
-   you're using a different database you'll also need to make sure the JDBC driver is available on
-   the Kafka Connect process's ``CLASSPATH``.
+.. include:: includes/prerequisites.rst
+    :start-line: 2
+    :end-line: 8
 
-Start by creating a database (you'll need to install SQLite if you haven't already)::
+.. include:: includes/prerequisites.rst
+    :start-line: 11
+    :end-line: 45
 
-   $ sqlite3 test.db
-   SQLite version 3.8.10.2 2015-05-20 18:17:19
-   Enter ".help" for usage hints.
-   sqlite>
 
-Next in the SQLite command prompt, create a table and seed it with some data::
+------------------------------
+Load the JDBC Source Connector
+------------------------------
 
-   sqlite> CREATE TABLE accounts(id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, name VARCHAR(255));
-   sqlite> INSERT INTO accounts(name) VALUES('alice');
-   sqlite> INSERT INTO accounts(name) VALUES('bob');
+Load the predefined JDBC source connector.
 
-Now we create a configuration file that will load data from this database. This file is included
-with the connector in ``./etc/kafka-connect-jdbc/source-quickstart-sqlite.properties`` and contains the
-following settings::
+#.  Optional: View available connectors with this command:
```

Comment:
If you change above, then please change here too. 

_View available predefined connectors with _